ktor-core/src/org/jetbrains/ktor/features/CallLogging.kt

```diff
@@ -35,24 +37,34 @@ class CallLogging(private val log: Logger, private val monitor: ApplicationMonit
         override val key: AttributeKey<CallLogging> = AttributeKey("Call Logging")
         override fun install(pipeline: Application, configure: Configuration.() -> Unit): CallLogging {
             val loggingPhase = PipelinePhase("Logging")
-            Configuration().apply(configure)
-            val feature = CallLogging(pipeline.log, pipeline.environment.monitor)
+            val configuration = Configuration().apply(configure)
+            val level = when {
+                configuration.level == Level.TRACE && pipeline.log.isTraceEnabled -> Level.TRACE
+                configuration.level == Level.DEBUG && pipeline.log.isDebugEnabled -> Level.DEBUG
+                configuration.level == Level.INFO && pipeline.log.isInfoEnabled -> Level.INFO
```

Comment:
Disagree with the code. As it is written, if I set logging level in code to TRACE, but it will be disabled in logging config (say, for production deployment), it will suddenly throw and fail to start application. This is not what is expected.